sys/include/net/nanocoap.h

```diff
@@ -585,7 +585,7 @@ ssize_t coap_opt_get_next(const coap_pkt_t *pkt, coap_optpos_t *opt,
  * @param[out]    value       start of the option value
  *
  * @return        length of option; 0 if the option exists but is empty
- * @return        -ENOENT if option not found
+ * @return        -ENOENT if option not found or option cannot be parsed
```

Comment:
Shouldn't this rather be a distinct error code? E.g. `-EINVAL`?

Reply:
Good idea. Changed.